### PR TITLE
Add Django 1.11 support

### PR DESCRIPTION
**Pull request by yakky**
No code is impacted by Django 1.11 and tests pass fine

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -15,9 +16,10 @@ deps =
     dj18: -rtest_requirements/django_1.8.txt
     dj19: -rtest_requirements/django_1.9.txt
     dj10: -rtest_requirements/django_1.10.txt
+    dj11: -rtest_requirements/django_1.11.txt
     cms32: django-cms>=3.2,<3.3
     cms33: django-cms>=3.3,<3.4
-    cms34: django-cms>=3.4,<3.5
+    cms34: django-cms>=3.4.5,<3.5
```
Comment (czpython) [inline tox.ini]: why hardcode 3.4.5 here?

Comment (yakky) [reply tox.ini]: To ensure proper 1.11 support (it might be an overkill, though)

Comment (czpython) [reply tox.ini]: Yeah I think is fine to leave it as 3.4 because once there's another 3.4.x release we might have to come back and change the ```.5``

Comment (yakky) [reply tox.ini]: Done

**Comment by merwok** [comment]
This looks great!  Any plans for a release on PyPI?

**Comment by ipmb** [comment]
While not such a big deal using straight `pip`, when using `pipenv`, the current release triggers an error if you try to create a `Pipfile.lock` with `Django>=1.11`.

```bash
$ cat Pipfile
[[source]]

url = "https://pypi.python.org/simple"
verify_ssl = true
name = "pypi"


[dev-packages]



[packages]

django = "==1.11.8"
aldryn-common = "*"
$ pipenv lock
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
```

**Pull request merged**